src/main/kotlin/org/jetbrains/research/kotlin/mpp/inference/operators/math/Add.kt

```diff
@@ -7,6 +7,6 @@ import org.jetbrains.research.kotlin.mpp.inference.tensors.Tensor
 class Add<T : Number> : Operator<T>() {
     override fun apply(inputs: Collection<Tensor<T>>): Collection<Tensor<T>> {
         require(inputs.size == 2) { "Applicable only for two arguments" }
-        return listOf((inputs.first() + inputs.last())!!)
+        return listOf(inputs.first() + inputs.last())
```

Comment:
..... If there were only two arguments.... Why do we recreate list? It will be equal to previous